_i18n/de/resources/moneropedia/mnemonicseed.md

```diff
@@ -1,19 +1,19 @@
 ---
-terms: ["mnemonic-seed", "mnemonic"]
-summary: "a 13 or 25 word phrase used to backup a Monero account, available in a number of languages"
+terms: ["mnemonic-seed", "mnemonic", "mnemonischer-Seed", "mnemonischen-Seed", "mnemonischen-Seeds"]
+summary: "Eine 13- oder 25-teilige Wortfolge, um ein Monero-Konto zu sichern; verfügbar in mehreren Sprachen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A 13 or 25 word phrase used to backup a Monero account, available in a number of languages. This 25-word phrase (13 words in the case of MyMonero) has all the information needed to view and spend funds from a Monero @account.
+Eine aus 13 oder 25 Wörtern bestehende Wortfolge, welche zur Sicherung eines Monero-Kontos genutzt wird und in mehreren Sprachen verfügbar ist. Diese 25-teilige Wortfolge (im Fall von MyMonero 13-teilig) beinhaltet alle benötigten Informationen, um Gelder eines Monero-@Kontos ansehen und ausgeben zu können.
 
-### In-depth Information
+### Ausführliche Informationen
 
-In the official wallet, the mnemonic seed comprises 25 words with the last word being used as a checksum. Those words correspond to a 256-bit integer, which is the account's *private* @spend-key. The *private* @view-key is derived by hashing the private spend key with Keccak-256, producing a second 256-bit integer. The corresponding *public* keys are then derived from the private keys.
+Beim offiziellen Wallet enthält der mnemonische Seed 25 Wörter, von welchen das letzte als Prüfsumme genutzt wird. Diese Wörter entsprechen einem 256-Bit-Integer, welcher der *private* @Spend-Key des Kontos ist. Der *private* @View-Key entsteht aus der Berechnung des privaten Spend-Keys mit Keccak-256, wodurch ein zweiter 256-Bit-Integer generiert wird. Die entsprechenden *öffentlichen* Schlüssel leiten sich von den privaten Schlüsseln ab.
```

Comment:
True, I'm going to add your suggestion. I would just leave out the link to Wikipedia for now since I don't know if we can just add something out of the blue (although the link makes sense here). @erciccione

Reply:
I also think the link might be useful - but I mostly added it only to show that "Hashfunktion" can be used in German.